Dapper/SqlMapper.cs

```diff
@@ -463,7 +464,7 @@ public static void SetDbType(IDataParameter parameter, object value)
                         try
                         {
                             handler = (ITypeHandler)Activator.CreateInstance(
-                                typeof(SqlDataRecordHandler<>).MakeGenericType(argTypes));
+                                typeof(SqlDataRecordHandler<>).MakeGenericType(argTypes))!;

```

Comment:
Should this use `(ITypeHandler?)` instead?

Reply:
No, because Activator.CreateInstance returns non-null for this case. Probably just `Nullable<T>` when it doesn't.